### PR TITLE
handle release_summary from config

### PR DESCRIPTION
**Pull request by KB-perByte**

### CODE REVIEW

File: .github/actions/ansible_validate_changelog/validate_changelog.py [hunk 1]
```diff
@@ -125,6 +125,8 @@ def is_valid_changelog_format(path: str) -> bool:
             changelog_config = yaml.safe_load(config_file)
             changes_type = tuple(item[0] for item in changelog_config["sections"])
             changes_type += (changelog_config["trivial_section_name"],)
+            changes_type += (changelog_config["prelude_section_name"],)
+            changes_type += (changelog_config["new_plugins_after_name"],)
```
Comment (NilashishC) [inline .github/actions/ansible_validate_changelog/validate_changelog.py:129]: What does this do?

Comment (NilashishC) [reply .github/actions/ansible_validate_changelog/validate_changelog.py:129]: Well, I mean I kinda get what it's doing, I just don't understand the key name here. Might be beyond the scope of this PR though.

Comment (KB-perByte) [reply .github/actions/ansible_validate_changelog/validate_changelog.py:129]: That is correct, I am not sure what section should be `prelude_section_name`
I"ll take it out!

Comment (NilashishC) [reply .github/actions/ansible_validate_changelog/validate_changelog.py:128]: If you take prelude section name out, it wouldn't recognize `release_summary`.

Comment (KB-perByte) [reply .github/actions/ansible_validate_changelog/validate_changelog.py]: @NilashishC My bad, removed `new_plugins_after_name`  now, confused it with `prelude_section_name` while removing did not check config.yml 😿

**Review by NilashishC — APPROVED** [review]

**Pull request merged**